README.rst

```diff
@@ -32,21 +32,81 @@ Javascript. With Batavia, you can run Python bytecode in your browser.
 It honors Python 3.4.4+ syntax and conventions, and let's you
 reference objects and classes defined natively in JavaScript.
 
+Getting Started
+---------------
+
+Getting a working local copy of Batavia requires a few steps: getting a copy of
+the Batavia code, and the ouroboros dependency within a virtual environment.
+
+You'll need to have Pyhton 3.4 available for Batavia to work. Instructions on
+how to set this up are `on our Environment setup guide
+<http://pybee.org/contributing/first-time/setup/>`_. 
+
+Setup a `pybee` folder to store everything::
+
+   $ mkdir pybee
+   $ cd pybee
+
+Get a copy of the Batavia code by running a :code:`git clone`::
+
+   $ git clone https://github.com/pybee/batavia
+
+Batavia requires a copy of Ouroboros (the Python standard library, written in
+Python) to build, so we also need to clone that::
+
+   $ git clone https://github.com/pybee/ouroboros
+
+Then, we need to create a virtual environment, and install Batavia into it.
+
+ * For Linux, MacOS::
+
+   $ virtualenv venv
+   $ . venv/bin/activate
+   $ cd batavia
+   $ pip install -e .
+
+ * For Windows::
+
+   > virtualenv --python=c:\python34\python.exe env
+   > cd env\Scripts
+   > activate
+   > pip install -e .
+
+ * For Anaconda users::
+
+   $ cd batavia
+   $ conda create -n batavia
+   $ source activate batavia
+   $ pip install -e .
+
+And finally, run the :code:`make` script to generate the combined Batavia JavaScript files::
+
+   $ make
+
+Your final setup should end up looking like this:: 
+
+  _ pybee
+    \_ batavia
+    \_ ouroboros
+    \_ venv
+
+You now have a working Batavia environment!
+
 Tutorial
 --------
 
-To take Batavia for a spin, run through the `Getting Started guide`_.
-
-Then have some fun with `the first tutorial`_, and try out running Python in
-your browser.
+Once you've setup your environment, you can have some fun with `the first
+tutorial`_, and try out running Python in your browser.
 
-.. _Getting Started guide: https://batavia.readthedocs.io/en/latest/intro/getting-started.html
 .. _the first tutorial: https://batavia.readthedocs.io/en/latest/tutorials/tutorial-0.html
```

Comment:
>  For a more detailed walkthrough, see the tutorial

This is a very good point (and sort of a 💡 moment for me). The tutorial should be the walkthrough. We should have a Quick Guide and a more Detailed Guide. A Tutorial is a good fit for the second version.